BrainPortal/app/controllers/demands_controller.rb

```diff
@@ -0,0 +1,487 @@
+
+require 'new_account_offerings'
+
+class DemandsController < ApplicationController
+
+  Revision_info=CbrainFileRevision[__FILE__] #:nodoc:
+
+  include DemandsHelper
+  before_filter :login_required,           :only => [:index, :multi_action, :approve, :destroy, :delete_multi, :clean_not_notified_multi, :resend_conf_multi, :fix_login_multi, :approve_multi, :approve_one, :resend_confirm]
+  before_filter :admin_role_required, :only => [:index, :multi_action, :approve, :destroy, :delete_multi, :clean_not_notified_multi, :resend_conf_multi, :fix_login_multi, :approve_multi, :approve_one, :resend_confirm]
+
+
+  def show
```

Comment:
Add the `#:nodoc:` for standard actions that do standard things; document the others